skypy/galaxy/tests/test_spectrum.py

```diff
@@ -173,11 +173,16 @@ def test_template_spectra():
     np.testing.assert_allclose(mt, m)
 
     # Test distance modulus
-    redshift = np.array([0, 1, 2])
+    redshift = np.array([0.00001, 1, 2])
```

Comment:
```suggestion
    redshift = np.array([1, 2, 3])
```
Prefer simple values rather than a 'hack' to get something 'close' to redshift zero